extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -32,7 +32,7 @@ static public function onGetRailModuleList( &$modules ) {
 		return true;
 	}
 
-	public static function onBeforePageDisplay( OutputPage $out, Skin $skin ) {
+	public static function onBeforePageDisplay() {
```

Comment:
I think we should keep the hook function signatures as they are, even when the params are not used.